src/cache.jl

```diff
@@ -113,7 +113,7 @@ function updatecache(absentmodules::Vector{Symbol}, server)
     
     @async begin 
         str = readline(o)
-        data = base64decode(str)
+        data = base64decode(chomp(str))
```

Comment:
Alright, I think Jeff fixed this on ``master``, can you maybe confirm and then merge this?